user-guide/lobby-rules.md

```diff
@@ -50,11 +50,11 @@ please go to this thread. Banned players: Please keep personal feelings and atti
 posts. Be professional and clear, try to give any information possible. Unless you are the banned 
 user, banning admin, or another admin involved, we ask that you do not post unnecessarily.
 
-### Lobby and Bot Rules
+### Lobby, Bot and Private Host Rules
 
-Again the General Rules above apply to the Lobby as well to the bots. The bots are not private 
-rooms and the Admin are also moderating the bots. Admins have the right to take action upon you, 
-if you are found in breach of these rules.
+Again the General Rules above apply to the Lobby as well to the bots and private hosts.
+If you are using the website the Admins have the right to take action upon you.
```

Comment:
Nitpick, lobby, bot and private hosts are not a website. Perhaps we would be better off to explain that breaches of the rules can result in disciplinary actions including being banned from TripleA.

Reply:
Np edit away it was just a general concept.  I do agree that if peeps can fool the lobby display it is an engine problem.  Ill just keep a list of all mods in a new cat on the forum.  Edit as you wish

Reply:
@DanVanAtta  Since you asked in our chat I will list a few examples which have been discussed over the years.  I have no problem with it being public since it is common knowledge to most TripleA users.  Cernel would be a good example.  Cernel = CerneI =Cerne! =Cernel1  mainly please note the i is not defined from the l.  Additional points could be made about the mod brackets.  This has happened repeatedly in the past.  Just read the bunker or old git issues.  Thus why I think a list should be maintained.  Just my 2 cents do as you wish. Pras

Reply:
@prastle I'm not aware of either old bunker posts or git issues mentioning the problem of moderator spoofing. Perhaps if you have any mind you could PM bunker links to me?

Reply:
I don't keep track of it.  I do know it was mentioned in the hated link of old during the original 1.9 release but really you would just have to ask Cernel.  he can give you first hand info.  Either way I am done with this since you apparently need proof of any statement.  I already explained exactly what the player did.

Reply:
Just wanting background information @prastle as it might be available.